docs/user-guides/finetuner.md

```diff
@@ -92,6 +94,7 @@ run = finetuner.fit(
     learning_rate=1e-5,
     loss='CLIPLoss',
     cpu=False,
+    to_onnx=True,
```

Comment:
As finetuner supports open_clip, can we finetune `model='ViT-B-32::openai'` in this tutorial.

Reply:
this model name does not match that in finetuner